charset_normalizer/cd.py

```diff
@@ -82,6 +83,7 @@ def encoding_languages(iana_name: str) -> List[str]:
     return unicode_range_languages(primary_range)
 
 
+@lru_cache(maxsize=None)
```

Comment:
I am mitigated about this one. Is it really worth it?

Reply:
It's used in a language decoding loop in `models.py` which could be called several times IMHO, so why not?

Reply:
OK. Just that the function is really flat. I am not against it.